src/server/server.ts

```diff
@@ -154,7 +152,7 @@ const liveBlockUpdates = (since: Date, content: Content, context: Context): Live
 });
 
 const recentLiveBlocks = (content: Content, context: Context): LiveUpdates => ({
-    newBlocks: serialiseLiveBlocks(recentBlocks(10)(content)(context)),
+    newBlocks: serialiseLiveBlocks(recentBlocks(7)(content)(context)),
```

Comment:
I think the current app requests 10 when fetching blocks.

Reply:
Which request are you talking about here, we have a few 🙂. The "View More" request? The polling request? The pull-to-refresh?

Reply:
I think all of the requests that load more blocks fetch 10. I'm not sure why the initial 7 is different :)

Reply:
Also, it's different on dotcom, I think it varies depending on the piece 🤷‍♂️.

Anyway, as mentioned above this is currently replacing client-side the initial seven rendered on the server; if I make it 10 three new blocks will pop in 🙂. The way this works may change in future.